cicd/forgeops-tests/config/ProductConfig.py

```diff
@@ -76,7 +76,7 @@ def __init__(self):
         try:
             self.ig_url = os.environ['IG_URL']
         except KeyError:
-            self.ig_url = 'https://openig.smoke.forgeops.com/'
+            self.ig_url = 'https://openig.smoke.forgeops.com'
```

Comment:
perhaps for another time - but can we make tests so they work in any namespace / domain

Reply:
Ahh - nvm, I see you are getting this from the environment

Reply:
Yes, these are just defaults.